kafka-webview-ui/pom.xml

```diff
@@ -124,6 +124,11 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
+        <!-- Micrometer Prometheus registry  -->
+        <dependency>
+            <groupId>io.micrometer</groupId>
+            <artifactId>micrometer-registry-prometheus</artifactId>
```

Comment:
Should we pin this to a specific version?

Reply:
Everywhere I have seen this in use, no version was specified.
We can specify the lastest version 1.3.2.
Do you think that Spring Boot parent or Spring Boot Actuator has a sort of mechanism to identify and import the right micrometer version?

Reply:
Looks like you are correct that it's defined by the parent pom [here](https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-dependencies/pom.xml#L154).

Seems good to leave out the version then 👍

Reply:
That's great! Exactly as I thought! I was not able to find the specific file to have a look into versions!
Thanks, solving this.